<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginInfoWidget</class>
 <widget class="QWidget" name="LoginInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>370</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#LoginInfoWidget{background-color: rgb(85, 170, 255);
border-style: inset;
border-width:1px;border-color:#2C3E50;}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>291</width>
     <height>281</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Account</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Password</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);  
font-size: 15px;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Login Infomation:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>close</string>
   </property>
   <property name="styleSheet">
    <string notr="true">#CloseButton{border-radius:10px;color:#ffffff;font-size:10px;background-repeat:no-repeat;
background-image: url(:/images/images/CloseButton.png);}
#CloseButton:hover{
background-image: url(:/images/images/CloseButtonPress.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="MinButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>minimize</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">#MinButton{color:#ffffff;font-size:10px;border:none;
background-image: url(:/images/images/MinButton.png);background-repeat:no-repeat;
border-radius:10px;}
#MinButton:hover{background-image:url(:/images/images/MinButtonPress);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>174</x>
     <y>340</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>MinButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginInfoWidget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CloseButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginInfoWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
